### DISCUSSION TITLE
Zibgee2MQTT - Smart Motion Sensor EZ-P1 device - recognized as Water leak sensor

### DISCUSSION THREAD

**Discussion by Atanas-Hristov** [Koenkk/zigbee2mqtt#25152] [Devices]
Hello,

I am trying to figure out a strange problem.
FYI - I am new to Zigbee2MQTT, MQTT and Home Assistant.

I just received today 5 battery powered motion sensors, courtesy of Ali Express and they got recognized as Water leak sensor

Here is a screenshot from the Zigbee2MQTT web page, it looks like this:

![image](https://github.com/user-attachments/assets/ea095ec5-77d8-48e1-ba79-1f7d3297cb7f)

Do you believe this to be an issue of wrongly recognizing the device, or is the problem within the device itself?
Here is how the device box looks like:

![image](https://github.com/user-attachments/assets/b38b9899-89c0-4290-9a7f-21fcef4ac4ef)
![image](https://github.com/user-attachments/assets/55d6a0a0-a52b-454b-8e66-c51dfe4134a2)
![image](https://github.com/user-attachments/assets/ccf32896-8de7-431d-916c-ac3f74d2040d)

Let me know if you need me to provide any more data on the device or from Zigbee2MQTT

Thank you for any assistance you could provide.

**Comment by flca-exxoss** [comment]
https://github.com/Koenkk/zigbee2mqtt/issues/24588
